bundle.Dockerfile.konflux

```diff
@@ -0,0 +1,71 @@
+FROM registry.redhat.io/ubi9/toolbox:9.4 AS builder
+
+COPY bundle/manifests /manifests
+COPY bundle/metadata /metadata
+COPY bundle/tests/scorecard /tests/scorecard
+COPY --chmod=0550 bundle/render_templates.sh /render_templates.sh
+COPY LICENSE licenses/
+
+ARG CERT_MANAGER_OPERATOR_IMAGE=registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:ae920b93103c45444f32c9686e0f090137a7076a8cf4ed8e86b3173b1487e402 \
+    CERT_MANAGER_WEBHOOK_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:7528e0e0c2a913af856fefb59d0103d0445c8269f90ee9ea99dc921bc2123eba \
+    CERT_MANAGER_CA_INJECTOR_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:7528e0e0c2a913af856fefb59d0103d0445c8269f90ee9ea99dc921bc2123eba \
+    CERT_MANAGER_CONTROLLER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:7528e0e0c2a913af856fefb59d0103d0445c8269f90ee9ea99dc921bc2123eba \
+    CERT_MANAGER_ACMESOLVER_IMAGE=registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:2411c83dc032195c29ad35aa8d887fa9da9db1376eb8247f2607e6bde6568554 \
```

Comment:
cert-manager related images does not exist yet. Once a release is created through konflux, these references will be updated by the konflux bot.